### DISCUSSION TITLE
v7.2.7 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#18340] [General]
### Engine Updates and Fixes

- On Unix, explicitly terminate the native process during cleanup only if it's not running in background (#18280)
- Stop sending telemetry about `ApplicationType` (#18168)

### General Cmdlet Updates and Fixes

- Remove the 1-second minimum delay in `Invoke-WebRequest` for downloading small files, and prevent file-download-error suppression (#18170)
- Enable searching for assemblies in GAC_Arm64 on Windows (#18169)
- Fix error formatting to use color defined in `$PSStyle.Formatting` (#18287)

### Tests

- Use Ubuntu 20.04 for SSH remoting test (#18289)

### Build and Packaging Improvements

<details>

<summary>

<p>Bump .NET to version 6.0.402 (#18188)(#18290)</p>

</summary>

<ul>
<li>Update cgmanifest (#18319)</li>
<li>Fix <code>build.psm1</code> to find the required .NET SDK version when a higher version is installed (#17299) (#18282)</li>
<li>Update MSI exit message (#18173)</li>
<li>Remove XML files for min-size package (#18274)</li>
<li>Update list of PS team members in release tools (#18171)</li>
<li>Make the link to minimal package blob public during release (#18174)</li>
<li>Add XML reference documents to NuPkg files for SDK (#18172)</li>
<li>Update to use version 2.21.0 of Application Insights (#18271)</li>
</ul>

</details>

[7.2.7]: https://github.com/PowerShell/PowerShell/compare/v7.2.6...v7.2.7

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - 5BF4A470B717F57F3B0E1EF1F16C9B1EB390555995E0DBE64F264101C7D490C9
- powershell_7.2.7-1.deb_amd64.deb
  - 93612AA09171838E997C85B78F6FE42041FBC73741B9751F975B1E908D6E77F0
- powershell-7.2.7-1.cm.x86_64.rpm
  - CDCDDB60450387E3C74B66942ECB295A7514463A47FECEBC88526C7D35D94ADA
- powershell-7.2.7-1.rh.x86_64.rpm
  - FC7C5759BA5CD4F28291F187E60348D7EBCC76841786480CA78A6E42439A0FA0
- powershell-7.2.7-linux-alpine-x64.tar.gz
  - 1E2098067BCE21EC116745F75594D1BC92A15BB736845453E490D96F8C46129D
- powershell-7.2.7-linux-arm32.tar.gz
  - FEE76FB4C3EB4700E05D113E8FC5D29DB9CBE05ED6A2191A8F09E4476B8BDEDA
- powershell-7.2.7-linux-arm64.tar.gz
  - FC189519457046A148944B56213750E09313DE79970439AA621746CDB9C9E36D
- powershell-7.2.7-linux-x64.tar.gz
  - 4F2560857CEC6C795F9940407D9F154D3C01CD6BA52E1656E3F2A5B60E5FF1B0
- powershell-7.2.7-linux-x64-fxdependent.tar.gz
  - EB20808673869AC5DAB7A56CE68F7964EA07776D77892A789AE761787D2C2C7D
- powershell-7.2.7-osx-arm64.pkg
  - E6FEE26BFA6265F6F76B7FA5F37826DA1D2C194398D4738783C44EC34A158C6E
- powershell-7.2.7-osx-arm64.tar.gz
  - FE6E4024402D43A3E2D06D25D83536CBBF6951637C8A7CE7207FBBAD58F63BF0
- powershell-7.2.7-osx-x64.pkg
  - 098FBD99699DCD02EA177266C0310C783ECE987B605BA8CC6CB49FC828FDD167
- powershell-7.2.7-osx-x64.tar.gz
  - 3DB758838DE5286CD59FFF354C491C12B1594D2B6DDCF507FE7FC03C7F6CD43E
- PowerShell-7.2.7-win-arm32.zip
  - BB1AE930EBD5F18DE48A052D93F5A195DA86E5F2E28B13149D384FB865E8D60A
- PowerShell-7.2.7-win-arm64.zip
  - 626AED993F6F7906920A0FED1645EE988A86AF9700607A04ED219122DD7F4851
- PowerShell-7.2.7-win-fxdependent.zip
  - E32B946C6DEE8CA0EE7CAE60B61C27954239DF0C1A9B9060088D663B88BBB639
- PowerShell-7.2.7-win-fxdependentWinDesktop.zip
  - E627E5BF29A519A6FC1446CF00C8A4CFD2B94E1A73CAD9075F591BD6117CFA07
- PowerShell-7.2.7-win-x64.msi
  - 7445F2F711759FE7167FAE2329911F9800D087265D1BA720A8801FF775AC1B31
- PowerShell-7.2.7-win-x64.zip
  - 4686202E22EC153F4FEF0B4D7FB2D577A6B0CD4D448B598E151417D095BC8FAD
- PowerShell-7.2.7-win-x86.msi
  - D9BAF2076D9F61BBB706B61542A68168FA8C870FFC41630780ACA3D1405444BD
- PowerShell-7.2.7-win-x86.zip
  - C82C042E05BA7FC66DD84D43FA46B33131ACFC7028B8AF724AC149171D2D24B5
- powershell-lts_7.2.7-1.deb_amd64.deb
  - DC85567C9A52E16EBED727F41389DE0B8E3275437E4B7A3905BC894F359A24F9
- powershell-lts-7.2.7-1.cm.x86_64.rpm
  - C41A1B01E0799EE03C95425D36CA6442A52D746956D7547D536322484A13F857
- powershell-lts-7.2.7-1.rh.x86_64.rpm
  - D14502B9F91649C49294A6978231EA1235804E443D3582A9276245160A875210
- powershell-lts-7.2.7-osx-arm64.pkg
  - AE99BC06DEDDDE939FA5130466FEF2B44B9460DA6C4198BB190C8BBD349F82C8
- powershell-lts-7.2.7-osx-x64.pkg
  - F819E17DE7D389C423BAB44C93C59EA132375D53F7077647AF7C79464AA3DB24

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.2.7'>v7.2.7 Release of PowerShell</a>.</em>

**Comment by def750** [comment]
when you see it

**Reply by tochikana** [reply]
ah yes, osu players bringing their most unfunny overused meme everywhere

**Reply by edgii** [reply]
when you see it

**Comment by bviktor** [comment]
Please publish to winget store as well, because PS already nags me to upgrade, but winget says everything's up to date...

**Reply by infinitepower18** [reply]
It's already on the winget repo, but the winget cli still shows the older version. Probably winget is messed up right now.

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/44692189/197400791-0b0d291c-7997-4398-a39a-5abb2dcb4566.png">

**Comment by m0xsec** [comment]
Some of the installer packages are being flagged by antivirus. For example, Google's Advanced Protection is blocking the MSI downloads in Google Chrome. Virus Total is also listing one AV engine that is flagging the Windows x64 MSI as a trojan.

https://www.virustotal.com/gui/file/7445f2f711759fe7167fae2329911f9800d087265d1ba720a8801ff775ac1b31?nocache=1

Can you verify that the latest build has not been tampered with? Are these detections benign?

**Reply by m0xsec** [reply]
I spotted checked some of the other files, they are also being flagged as malicious. Hashes appear to match what is listed in the release information.
